web/src/stream_data.js

```diff
@@ -581,10 +581,6 @@ export function can_unsubscribe_others(sub) {
         return false;
     }
 
-    if (page_params.is_admin) {
-        return true;
-    }
```

Comment:
(I think it's potentially OK to offer a "nobody" option here -- since under that definition, this is the group of who in addition to administrators has this permission)

Reply:
I was going to ask the question. Was waiting for CI to pass and then got away from computer for some time.

I am not sure whether we should show "nobody" option here since we do not mention about the implicit permission for admin in the label as well and it might confuse users.

I have not done any changes yet related to this in the PR. Will do once I have decided.

Reply:
Yeah, it's possible we should just leave "nobody" off the options offered in the UI until we can explain how it works better.